pkg/common/configs/configs.go

```diff
@@ -26,11 +26,34 @@ import (
 )
 
 const (
-	HealthCheckInterval = "health.checkInterval"
+	// prefixes
+	PrefixEvent  = "event."
+	PrefixHealth = "health."
+
+	HealthCheckInterval = PrefixHealth + "checkInterval"
+
+	// events
+	CMEventRequestsEnabled = PrefixEvent + "requestsEnabled"
+	CMEventTrackingEnabled = PrefixEvent + "trackingEnabled"
+	CMEventRequestCapacity = PrefixEvent + "requestCapacity"
+	CMEventBufferCapacity  = PrefixEvent + "bufferCapacity"
+	CMEventResponseSize    = PrefixEvent + "responseMaxSize"
```

Comment:
I think we can get rid of `responseMaxSize` as well -- we can drive DoS protection via request capacity and buffer capacity just fine.

Reply:
PR updated with the changes.